karl/views/batch.py

```diff
@@ -355,6 +355,38 @@ def get_fileline_batch(fp, context, request, batch_start=0, batch_size=20,
     _add_link_data(info, context, request)
     return info
 
+def get_simple_batch(results, context, request, batch_start=0, batch_size=20):
```

Comment:
Is this only used here: https://github.com/karlproject/karl/pull/127/files#diff-c567c69f3a25169259ab642a4064a933R109? 

Could be simplified a lot with SQLAlchemy, when ``results`` would be a query object (untested code, just for illustration purposes):

```
  entries = results.offset(batch_start).limit(batch_size)
  total = results.count()
```

Let's also postpone this, but keep in mind.